eo-phi-normalizer/test/eo/phi/rules/yegor.yaml

```diff
@@ -93,6 +93,18 @@ rules:
       - nf: ["!n"]
     tests: []
 
+  - name: Rule 10
+    description: ""
+    pattern: ⟦ !t ↦ !b1, !B1 ⟧(!t ↦ !b2, !B2)
+    result: ⊥
+    when:
+      - not_equal: ["!t", "ρ"]
```

Comment:
Somehow I do not like using a tuple here, but I can't come up with a better alternative :)